### ISSUE TITLE
Sidebar documentation unclear

### ISSUE THREAD

**Issue by JunaidQadirB** [VanOns/laraberg#23]
I'm following the [documentation](https://github.com/VanOns/laraberg#select) to add a `select` to the [sidebar](https://github.com/VanOns/laraberg#sidebar) but Its not working as expected. 

![Screenshot from 2019-05-23 20-49-39](https://user-images.githubusercontent.com/642353/58268037-21fc5300-7d9e-11e9-9439-03ff5d57f471.png)

What am I doing wrong?

**Comment by mauricewijnia** [comment]
I'd like to help you, but you're not providing enough information for me to do so...
Can you please describe your issue using this template?
Please add your `Laraberg.init` call and your sidebar html as well :)

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Environment (please complete the following information):**
- Laraberg Verison: [e.g. 1.0.0]
 - OS: [e.g. iOS]
 - Browser [e.g. chrome, safari]
 - Version [e.g. 22]


**Additional context**
Add any other context (like code fragments) about the problem here.

**Comment by JunaidQadirB** [comment]
I want to add a field to the Laraberg sidebar, a select in this case.

I want to add a field to the Laraberg sidebar, a select in this case.

**Steps To Reproduce**

1. Add a `textarea` tag with an `id` attribute of `body`

```html
<textarea id="body" name="body" rows="50" hidden></textarea>
```

2. Initialize Laraberg with `sidebar` option set to `true`

```javascript
document.addEventListener('DOMContentLoaded', 		function () {
    	Laraberg.init('body',
        {
            minHeight: '500px',
            laravelFilemanager: true,
            sidebar: true,
	})
})
```

3. Add `select` markup inside a `div` with class name `laraberg-sidebar`

```html
<div class="laraberg-sidebar">
  <label for="article-month">Month</label>
  <select id="article-month" name="month" class="uk-select uk-form-large ">
      <option value="1">January</option>
      <option value="2">February</option>
      <option value="3">March</option>
      <option value="4">April</option>
  </select>
</div>
```

5. Reload the page

**Expected Behavior**

The `select` field should appear in the sidebar of Laraberg

**Current Behavior**

The `select` dropdown appears outside Laraberg just like a regular `select` would render on the page. 

![Screenshot from 2019-05-23 20-49-39](https://user-images.githubusercontent.com/642353/58268037-21fc5300-7d9e-11e9-9439-03ff5d57f471.png)


**Env/Version Info**

Laravel: `5.8.*`

Laraberg: `0.0.3@beta`

OS: `Ubuntu 19.04`

Browsers: Firefox, Chrome

**Comment by mauricewijnia** [comment]
This feature is available from laraberg 0.0.4 and higher, so you should update.
